src/dpq2/conv/to_d_types.d

```diff
@@ -204,7 +231,7 @@ public void _integration_test( string connParam ) @system
         C!PGreal(-12.3456f, "real", "-12.3456");
         C!PGdouble_precision(-1234.56789012345, "double precision", "-1234.56789012345");
         C!PGtext("first line\nsecond line", "text", "'first line\nsecond line'");
-        C!PGtext("12345 ", "char(6)", "'12345'");
+        C!PGtext("12345 ", "char(6)", "'12345 '");
```

Comment:
Can you explain that?

Reply:
I've modified this because of the other way around test. But you are right that it broken the original meaning of test. So I altered the other way around test to accept it with the original value.